crml/cennzx-spot/rpc/runtime-api/src/lib.rs

```diff
@@ -49,5 +50,15 @@ sp_api::decl_runtime_apis! {
 			amount: Balance,
 			asset_to_buy: AssetId,
 		) -> CennzxSpotResult<Balance>;
+		/// Query the value of liquidity in the exchange for `asset_id` for `account`
+		fn liquidity_value(
+			account: AccountId,
+			asset_id: AssetId,
+		) -> (Balance, Balance, Balance);
```

Comment:
If the return type was a struct, it could be clearer what this function returns. However, if you prefer a tuple, I would suggest you add a documenting comment for the return values.

Reply:
Nice! I have added some comments 👍